scripts/multinode/configure-pri.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+set -euo pipefail
+
+OC_CMD="sudo -i oc --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig"
+KUBECTL_CMD="sudo -i kubectl --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig"
+
+PRI_NODE_HOST=
+PRI_NODE_ADDR=
+SEC_NODE_HOST=
+SEC_NODE_ADDR=
+
+function usage() {
+    echo "Usage: $(basename "$0") <primary_host_name> <primary_host_ip> <secondary_host_name> <secondary_host_ip>"
+    exit 1
+}
+
+function configure_system() {
+    # Disable selinux
+    # TODO: remove once selinux is working properly again
+    sudo setenforce 0
+
+    # TODO: Edit firewall rules instead of stopping firewall
+    sudo systemctl stop firewalld
+    sudo systemctl disable firewalld
+
+    # Configure the MicroShift host name
+    sudo hostnamectl hostname "${PRI_NODE_HOST}"
+
+    # Update /etc/hosts to resolve primary and secondary host names
+    if ! grep -q "${PRI_NODE_HOST}" /etc/hosts ; then
```

Comment:
Only add something to /etc/hosts if they are not already resolvable.

Reply:
This turned out a bit tricky to implement because `getent` output parsing may not be trivial. 
Since there's no harm adding this statement to /etc/hosts if it does not already exist, can I suggest keeping the code as-is?
```
$ getent hosts microshift-rpm-pri
fe80::5054:ff:fe3a:bc39 microshift-rpm-pri
fe80::42:adff:fe49:a0a5 microshift-rpm-pri
fe80::821:30ff:fee1:a3ea microshift-rpm-pri
fe80::891:deff:feca:fd47 microshift-rpm-pri
fe80::dc2f:cfff:fefd:c664 microshift-rpm-pri
fe80::8492:f7ff:fe05:6af0 microshift-rpm-pri
fe80::6022:e1ff:fe69:7bef microshift-rpm-pri
fe80::3419:47ff:fec0:190a microshift-rpm-pri
fe80::e4b8:59ff:fea9:be2e microshift-rpm-pri
fe80::684e:4dff:fe8d:86c6 microshift-rpm-pri
fe80::7c79:74ff:fe4d:3352 microshift-rpm-pri
```

Reply:
Try `getent ahostsv`:

[microshift@microshift-node-may-10 scripts]$ getent ahostsv4 microshift-node-may-10 | head -1 | awk '{ print $1 }'
192.168.122.222
[microshift@microshift-node-may-10 scripts]$ getent ahostsv4 not_resolvable
[microshift@microshift-node-may-10 scripts]$ echo $?
2

Reply:
That worked, thank you.